themes/navy/layout/partial/plugin.swig

```diff
@@ -1,9 +1,10 @@
 <li class="plugin on">
+  <img src="/plugins/thumbnails/{{ plugin.thumbnail }}" />
```

Comment:
Maybe needs an `if`? In case there is not thumbnail, or we just always put one?

Reply:
we should always put one (even if a default thumbnail)